modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-test/src/testIntegration/java/com/liferay/portal/workflow/metrics/rest/resource/v1_0/test/SLAResourceTest.java

```diff
@@ -212,7 +216,14 @@ protected SLA testPutSLA_addSLA() throws Exception {
 
 	private void _deleteSLAs() throws Exception {
 		for (SLA sla : _slas) {
-			slaResource.deleteSLA(sla.getId());
+			_workflowMetricsSLADefinitionLocalService.
```

Comment:
I don't understand this code, but the name of the method does not match with what you are doing inside, I think

Reply:
I've revised it and apparently thats how the delete SLA Definition method is called in Workflow Metrics. We had to use this becouse if we used the delete method of the REST APIs here, after the delete methods were executed a Not Found exception was thrown becouse of this.